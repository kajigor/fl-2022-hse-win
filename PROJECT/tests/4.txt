<S> = '(' <A> ')'<B>'{';
<B> = 'E';
<B> = <B>'b';
<A> = 'a';
