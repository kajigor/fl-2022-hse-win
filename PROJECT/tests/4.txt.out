LexToken(START_NON_TERM,'S',1,0)
LexToken(EQ,'=',1,4)
LexToken(TERM,'(',1,6)
LexToken(NON_TERM,'A',1,10)
LexToken(TERM,')',1,14)
LexToken(NON_TERM,'B',1,17)
LexToken(TERM,'{',1,20)
LexToken(END,';',1,23)
LexToken(NON_TERM,'B',2,0)
LexToken(EQ,'=',2,4)
LexToken(EPSILON,'E',2,6)
LexToken(END,';',2,9)
LexToken(NON_TERM,'B',3,0)
LexToken(EQ,'=',3,4)
LexToken(NON_TERM,'B',3,6)
LexToken(TERM,'b',3,9)
LexToken(END,';',3,12)
LexToken(NON_TERM,'A',4,0)
LexToken(EQ,'=',4,4)
LexToken(TERM,'a',4,6)
LexToken(END,';',4,9)
